WPFClient/Pages/ChatPage.xaml.cs

```diff
@@ -94,20 +80,21 @@ public void DisplayUsers()
         {
             App._connection.On("UpdateUsersList", (List<string> userNames) =>
             {
-                foreach(var localUser in Users){
-                    if(!userNames.Contains(localUser))
+                foreach (var localUser in Users)
```

Comment:
These variables, i.e. localUser and remoteUser, should be renamed to be more clear on what they are.

Reply:
Woooooow